src/main/java/org/jboss/xavier/integrations/jpa/service/WorkloadInventoryReportService.java

```diff
@@ -49,6 +49,23 @@
         return reportRepository.findByAnalysisOwnerAndAnalysisId(analysisOwner, analysisId);
     }
 
+    public List<WorkloadInventoryReportModel> findByAnalysisOwnerAndAnalysisId(
+            String analysisOwner,
+            Long analysisId,
+            SortBean sortBean,
+            WorkloadInventoryFilterBean filterBean
+    ) {
+        // Sort
+        Sort.Direction sortDirection = sortBean.isOrderAsc() ? Sort.Direction.ASC : Sort.Direction.DESC;
```

Comment:
If we dont specify anything it will fall-back to DESC sorting. Is that what it is expected ? I would expect an ASC sorting by default

Reply:
@jonathanvila I've read the documentation from Miguel and what is required is to have Default order by: `Provider, Datacenter, Cluster, VM name`. I've made some changes to be consistent with what is required from the Document.

https://docs.google.com/document/d/1jTPrwKFdVIO7bkhNLA6skIneQYBmReaJQDlJW9SoZZk/edit#